torch_tvm/compiler.cpp

```diff
@@ -55,7 +55,10 @@ tvm::relay::Var TVMCompiler::convertToRelay(Value* val, TVMContext ctx) {
       "Expected CPU device type but got:", device_type);
     tvm::Array<tvm::relay::IndexExpr> sizes;
     const auto& varying_sizes = pt_t->sizes();
-    for (const auto& optional_size : varying_sizes.sizes()) {
```

Comment:
Yeah, it's weird that I had met with `use out of scope` ASAN error if I just chain the calls together.